styles.css

```diff
@@ -50,3 +50,11 @@ ol {
 #slide-shapes-transitions-image img {
   height: 25rem;
 }
+
+.slidev-layout {
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
In order to make an element scrollable on y axis, the height of the direct parent needs to be smaller than the child(ren).

The quick fix was the initial solution where I made the whole container scrollable, and it worked because the slide content had a fixed height set by slidev, and the text made the children overflow because of larger height than the container.
![Screenshot 2023-08-16 at 09 28 12](https://github.com/nearform/the-fastify-workshop/assets/141126446/7e6cf3f9-5235-47a0-81f3-681c24d53293)


Because we wanted only the code to be scrollable, and keep the heading in place even if scrolling, I had to make the container of the code to be smaller than the code content itself, because there wasn't any restriction set on its height:
![Screenshot 2023-08-16 at 10 04 47](https://github.com/nearform/the-fastify-workshop/assets/141126446/c2ccee81-5498-4ed1-80cb-7fde9e39057f)

We want the above element to take only the space available, and not overflow. Setting the parent to `display: flex` organises children that way, so the desired situation is given. The container of the code is only taking the available space in the parent container, while the code itself in its container is higher than the container:
![Screenshot 2023-08-16 at 10 08 20](https://github.com/nearform/the-fastify-workshop/assets/141126446/893bfc75-6c03-41ea-b6f8-1887aac305ae)

![Screenshot 2023-08-16 at 10 08 43](https://github.com/nearform/the-fastify-workshop/assets/141126446/7b89d685-f77d-40e5-bfab-9b3cca9ebffc)

The real goal by setting display to flex was only to be able to control the children's dimensions in the container.
I've tested all slides, haven't found any issues, and I doubt it would cause any, but definitely worth testing for others as well.